tests/utils/common/common.py

```diff
@@ -223,7 +223,17 @@ def determine_active_passive_part(bitbake_variables, conn):
 
 
 def get_ssh_common_args(conn):
-    args = "-O -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+    args = "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+
+    # Check if -O is supported. If it's not supported, we don't need it, because
+    # it is already the default.
+    output = subprocess.check_output(
+        ["scp", "--help"], stderr=subprocess.STDOUT
```

Comment:
Actually, `scp` seems to not support `--help`. I get:
 ```
 $ scp --help
scp: unknown option -- -
usage: scp [-346ABCOpqRrsTv] [-c cipher] [-D sftp_server_path] [-F ssh_config]
           [-i identity_file] [-J destination] [-l limit]
           [-o ssh_option] [-P port] [-S program] source ... target
```

So we can very well just do
```suggestion
        ["scp"], stderr=subprocess.STDOUT
```

Reply:
Actually, have you see the latest failures in your/mine PRs?
```
input = None, capture_output = False, timeout = None, check = True
popenargs = (['scp', '--help'],), kwargs = {'stderr': -2, 'stdout': -1}
process = <subprocess.Popen object at 0x7f70e82c0a30>
stdout = b'unknown option -- -\r\nusage: scp [-346BCpqrTv] [-c cipher] [-F ssh_config] [-i identity_file]\n            [-J destination] [-l limit] [-o ssh_option] [-P port]\n            [-S program] source ... target\n'
stderr = None, retcode = 1
...
...
>               raise CalledProcessError(retcode, process.args,
                                         output=stdout, stderr=stderr)
E               subprocess.CalledProcessError: Command '['scp', '--help']' returned non-zero exit status 1.
```